.gitignore

```diff
@@ -149,3 +149,6 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+# Mac
+.DS_Store
```

Comment:
Can you undo this change? User-specific ignores should be in the global gitignore, not the project gitignore.